CONTRIBUTING.md

```diff
@@ -0,0 +1,5 @@
+Hello there. We follow the [contribution guidelines] from the [nteract]
+project.
+
+[nteract]: https://nteract.io
+[contribution guidelines]: https://github.com/nteract/nteract/blob/master/CONTRIBUTING.md
```

Comment:
I wonder if we need to make a python version of this since we have a bunch of JS material in there too.

Reply:
Probably at some point. It may be good just to have a CONTRIBUTING that has basic courtesy stuff at the top and the code stuff following.